html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html" method="post">
+      <label for="email">Email
+        <div>
+          <input type="text" name="email" id="email" />
+        </div>
+      </label>
+      <label>Why is coding cool?
+        <div>
+          <select name="choices">
+            <option value="Select an option">Select an option</option>
+            <option value="Because it's interesting">Because it's interesting</option>
+            <option value="Because it's fun">Because it's fun</option>
+            <option value="Because it's powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </label>
+      <br>
+      <label for="language">Favorite Language (choose one)
```

Comment:
This is actually just general text describing the section below since it isn't labeling one specific form control element. You should use a `p` element.